openmetadata-ui/src/main/resources/ui/src/components/DatasetDetails/DatasetDetails.component.tsx

```diff
@@ -798,13 +797,7 @@ const DatasetDetails: React.FC<DatasetDetailsProps> = ({
               </Card>
             )}
             {activeTab === 8 && Boolean(dataModel?.sql) && (
-              <Card className="m-y-md h-full">
-                <SchemaEditor
-                  className="tw-h-full"
-                  mode={{ name: CSMode.SQL }}
-                  value={dataModel?.sql || ''}
-                />
-              </Card>
+              <DbtTab dataModel={dataModel} />
```

Comment:
We now have fallback to rawSQL so in above condition we can add that too?
Boolean(dataModel?.sql || dataModel?.rawSQL)

Reply:
nice catch, thanks!